config-model/src/main/java/com/yahoo/vespa/model/content/GlobalDistributionValidator.java

```diff
@@ -22,9 +22,11 @@
 
     public void validate(Map<String, NewDocumentType> documentDefinitions,
                          Set<NewDocumentType> globallyDistributedDocuments,
-                         Redundancy redundancy) {
-
-        verifyGlobalDocumentsHaveRequiredRedundancy(globallyDistributedDocuments, redundancy);
+                         Redundancy redundancy,
+                         boolean enableMultipleBucketSpaces) {
+        if (!enableMultipleBucketSpaces) {
+            verifyGlobalDocumentsHaveRequiredRedundancy(globallyDistributedDocuments, redundancy);
+        }
         verifySearchableCopiesIsSameAsRedundancy(globallyDistributedDocuments, redundancy);
```

Comment:
Shouldn't this check also be ignored when multiple bucket spaces are enabled? The global distribution config transform will ensure that number of ready copies == redundancy == number of nodes, but this property will not necessarily hold for the node group(s) specified in the XML.